pkg/commands/clusterstack/patch.go

```diff
@@ -29,25 +30,26 @@ type ImageRelocator interface {
 	Relocate(writer io.Writer, image v1.Image, dest string) (string, error)
 }
 
-func NewUpdateCommand(clientSetProvider k8s.ClientSetProvider, rup registry.UtilProvider, newWaiter func(dynamic.Interface) commands.ResourceWaiter) *cobra.Command {
+func NewPatchCommand(clientSetProvider k8s.ClientSetProvider, rup registry.UtilProvider, newWaiter func(dynamic.Interface) commands.ResourceWaiter) *cobra.Command {
 	var (
 		buildImageRef string
 		runImageRef   string
 		tlsCfg        registry.TLSConfig
 	)
 
 	cmd := &cobra.Command{
-		Use:   "update <name>",
-		Short: "Update a cluster stack",
-		Long: `Updates the run and build images of a specific cluster-scoped stack.
+		Use:     "patch <name>",
```

Comment:
As you mentioned, I think moving "update" to be an alias is the right direction, like this.